File: app/src/main/java/com/themaker/fshmo/klassikaplus/data/web/dto/PhotoDto.java
package com.themaker.fshmo.klassikaplus.data.web.dto;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PhotoDto implements Serializable
{

@SerializedName("id")
@Expose
private String id;
@SerializedName("image")
@Expose
private String image;
private final static long serialVersionUID = -9032062017948970785L;

public String getId() {
return id;
}

public void setId(String id) {
this.id = id;
}

public String getImage() {
return image;
}

public void setImage(String image) {
this.image = image;
}

}

File: app/src/main/java/com/themaker/fshmo/klassikaplus/data/web/dto/catalog/items/PhotoDto.kt
package com.themaker.fshmo.klassikaplus.data.web.dto.catalog.items

import java.io.Serializable

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class PhotoDto(
    @SerializedName("id")
    @Expose
    val id: String?,
    @SerializedName("image")
    @Expose
    val image: String?

) : Serializable {


    companion object {
        private const val serialVersionUID = -9032062017948970785L
    }

}